rocky/katalogus/utils.py

```diff
@@ -5,4 +5,6 @@
 
 
 def get_enabled_boefjes_for_ooi_class(ooi_class: type[OOI], organization: Organization) -> list[Boefje]:
-    return [boefje for boefje in get_katalogus(organization.code).get_enabled_boefjes() if ooi_class in boefje.consumes]
+    katalogus = get_katalogus(organization.code)
```

Comment:
Could we provide a property on the view that provides us with the katalogus?
This property can be added in the Organization Mixin, and uses the user to instantiate the Katalogus based on their memberships / rights and the organization from the request/url.

Reply:
Yes